setup.py

```diff
@@ -3,22 +3,29 @@
 
 # pylint: skip-file
 
+import os
 import sys
 from pathlib import Path
 
 from pkg_resources import parse_version
 from setuptools import find_packages, setup
 
 is_py37 = sys.version_info.major == 3 and sys.version_info.minor == 7
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'  # copied from the docs
```

Comment:
I followed this workflow: https://docs.readthedocs.io/en/stable/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules

Reply:
would it be useful to put that link into the comment?

Reply:
(I doubt anyone's going to manage to find these PR comments in the future)